src/bootstrap.styles.loader.js

```diff
@@ -0,0 +1,22 @@
+/* eslint func-names: 0 */
+import loaderUtils from 'loader-utils';
+
+import config from './bootstrap.config';
+import createBootstrapImport from './utils/createBootstrapImport';
+
+module.exports = function() {
+  if (this.cacheable) this.cacheable();
+
+  const { bootstrapPath } = loaderUtils.parseQuery(this.query);
+
+  // const useFlexbox = config.flexbox;
```

Comment:
In Bootstrap 4 we must define do we want to use flexbox model or not. This configuration is WIP.